packages/db/prisma/schema.prisma

```diff
@@ -81,6 +99,62 @@ model PreApprovedUser {
     @@map("pre_approved_users")
 }
 
+// Group model - collection of users with ownership and management
+model Group {
+    id          Int      @id @default(autoincrement())
+    name        String
+    description String?
+    created_at  DateTime @default(now())
+    updated_at  DateTime @updatedAt
+
+    // Group ownership
+    owner    User @relation("GroupOwner", fields: [owner_id], references: [id], onDelete: Cascade)
```

Comment:
I don't think we want delete user to delete their groups. I think admin should be forced to transfer ownership or manually delete groups.

Reply:
Yeah could change that - seems like a niche risk but agree the more risk averse approach is to block it to force us to handle that case